pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.usort]
+
+categories = [
+    "future",
+    "standard_library",
+    "third_party",
+    "pytorch",
+    "first_party",
```

Comment:
I've split the imports a little further. This way `torch` will always get imported before we import `torchvision`, but after the standard library and other third-party packages. If we later add dependencies to other PyTorch derivatives, we can simply add another section for them to make sure they are also imported after `torch`.

Reply:
> Does that mean we need to always import torch before other PyTorch derivatives like torchvision or does it even need to be imported before all other third party packages?

I think importing torch before torchvision (and other torchxyz) stuff should be enough. But who knows what `import torch` does... I've seen some weird stuff going on. In anycase this should be enough, if it's not, tests will hopefully fail to let us know